### DISCUSSION TITLE
Storage, HOME directory, etc. in NeuroDesktop

### DISCUSSION THREAD

**Discussion by TomEmotion** [neurodesk/neurodesk.github.io#127] [General]
I searched the docs a little for this but couldn't find so...

How is default filesystem storage determined/set when installing NeuroDesktop? Where is the user's HOME directory mounted by default and can this be changed?

When allocating Nectar resources (but also perhaps with some people's own latops) storage might be an issue. And as much as I hate it, various pipelines and software packages use the user's HOME directory for storage not only of settings, but other things too (atlases, imaging files, software packages/conda environments).

This might be a particular concern with conda environments.

Also might be worth discussing in the context of multiple users using the same NeuroDesktop instance for collaborative work.

Thanks.

**Comment by stebo85** [comment]
Currently the homedirectory of the user is not kept. The only persistent storage is /neurodesktop-storage. However, we are currently working on moving home-directory content to this persistent storage exactly for the use-cases that you describe.

@aswinnarayanan: once miniconda is gone from the homedirectory should we attempt at making the whole homedirectory persistent?

**Comment by TomEmotion** [comment]
That would be terrific. Would also make user interface settings (e.g. for editors, fonts, different software) and search paths persistent. We are happy to help with this if pointed in the right direction.

**Reply by stebo85** [reply]
ok, great. We finish up the migration of python and then tackle the persistent homedirectory
